src/CodiceFiscale/Calculator.php

```diff
@@ -103,9 +105,9 @@ private function calculateName()
     /**
      * Calculate small string for the given parameters (used by name and surname).
      *
-     * @param $consonants A consonants string.
-     * @param $string The small string.
-     * @returns Returns the calculated result for the small string.
+     * @param string $consonants   A consonants string.
+     * @param string $string  The small string.
+     * @return string Returns the calculated result for the small string.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.